lib/auto-scenario-config-for-ember.js

```diff
@@ -33,6 +33,9 @@ async function generateScenariosFromSemver(semverStatements, availableVersions)
             devDependencies: {
               'ember-source': versionNum,
             },
+            overrides: {
```

Comment:
These configs are used whether npm, yarn, or now pnpm is used. What does `overrides` do in `pnpm`?  Does `pnpm` also support the `$ember-source` syntax? 

Is it ignored properly in `yarn`? Does a new project in yarn have the same problem and need `resolutions`?

Reply:
so yarn@1.x never did strict peer checks

pnpm only listens to overrides that are inside a pnpm keyed object, and pnpm doesn't have the exact same issue because pnpm considers pre-releases as matching when their ranges match

Reply:
I wanted to confirm that both pnpm and yarn will ignore `overrides`, if so, I think this approach is fine for the automated config, but if we put it in the blueprint I think it will be confusing to non-npm users, so it should be conditional and only there for npm users.

Reply:
Will this hide peer dependency issues? For example, in the run up to 5.0, ember-try runs for addons caught that various other addons had incorrect (or overly restrictive) peer dependency on ember-source and needed to be loosened.

Reply:
this will only peer dependency issues with ember-source and only on auto-generated configs. When I say that I plan to add this to the default blueprint I mean only for release, canary, and beta because they will never work because of npm considering the pre-releases not inside legit ranges 😫

Reply:
beta and canary I understand, release should work though